docs/remote_jobs.rst

```diff
@@ -25,14 +25,31 @@ The `ansible-runner worker` command accepts this stream, runs the playbook, and
 stream of the resulting job events and artifacts.
 This command optionally accepts the `--private-data-dir` option.
 If provided, it will extract the contents sent from `ansible-runner transmit` into that directory.
-If no `--private-data-dir` is given, then it will extract the contents to a temporary directory,
-which will be deleted at the end of execution.
-You can use the `--delete` flag to assure that files are deleted from a location specified by `--private-data-dir` as well.
 
 The `ansible-runner process` command accepts the result stream from the worker, and fires all the normal callbacks
 and does job event processing.  In the command above, this results in printing the playbook output and saving
 artifacts to the data dir.  The `process` command takes a data dir as a parameter, to know where to save artifacts.
 
+Private Data Directory Cleanup
+------------------------------
+
+When running `ansible-runner worker`, if no `--private-data-dir` is given,
+it will extract the contents to a temporary directory which is deleted at the end of execution.
+You can use the `--delete` flag in conjunction with `--private-data-dir` to assure that
+the provided directory is deleted at the end of execution.
+
+The following command offers out-of-band cleanup.
+
+    $ ansible-runner worker cleanup --file-pattern=/tmp/foo_*
```

Comment:
Do we need to consider any security (or oopsie) implications of this? What if someone used `--file-pattern=/`, either accidentally or on purpose?

Reply:
I agree with some oopsie validation, but I don't know of any good pattern to follow for that. Yeah, throwing an error if given `/` would be a good idea, but I don't have a quality list of places that we should error for.

Reply:
I added a little bit of validation. It's not going to prevent everything that could go wrong, but at least it's something.